File: Implemented Funs Using JS/In Array.js
// Implemented Functions Will To check  Number Exist in Array

function inArray(array, target) {
  for (let i = 0; i < array.length; i++) {
    if (array[i] === target) return true;
  }
  return false;
}

if (inArray([1, 2, 6, 8, 9], 8)) console.log("Yes target exist in array");
else console.log("No target exist in array");

File: Implemented Funs Using C++/In Array.cpp
#include<iostream>
#include<vector>


using namespace std;

// Implemented Functions Will To check  Number Exist in Array

bool in_Array(vector<int>array,int target){
    for(int i=0;i<array.size();i++){
        if(array[i]==target) return true;
    }
    return false;
}

int main()
{
    
    
 if(in_Array({1,2,3},3)) cout<<"Yes target exist in array"<<endl;
 else cout<<"No target exist in array"<<endl;

    return 0;
}
